### PR TITLE
[E129-MG < T1029-MG] Fix logic in fine grained permissions

### PR DESCRIPTION
**Pull request by BorisTasevski**

### CODE REVIEW

File: src/glue/auth_handler.hpp [hunk 1]
```diff
@@ -68,17 +68,8 @@ class AuthQueryHandler final : public memgraph::query::AuthQueryHandler {
...
-#ifdef MG_ENTERPRISE
-      ,
-      const std::vector<std::unordered_map<memgraph::query::AuthQuery::FineGrainedPrivilege, std::vector<std::string>>>
-          &label_privileges,
-
-      const std::vector<std::unordered_map<memgraph::query::AuthQuery::FineGrainedPrivilege, std::vector<std::string>>>
-          &edge_type_privileges
-#endif
-      ) override;
+  void DenyPrivilege(const std::string &user_or_role,
```
Comment (gitbuda) [inline src/glue/auth_handler.hpp:71]: Shouldn't this be deleted?

File: src/query/interpreter.hpp [hunk 2]
```diff
@@ -112,17 +112,7 @@ class AuthQueryHandler {
...
-#ifdef MG_ENTERPRISE
-      ,
-      const std::vector<std::unordered_map<memgraph::query::AuthQuery::FineGrainedPrivilege, std::vector<std::string>>>
-          &label_privileges,
-
-      const std::vector<std::unordered_map<memgraph::query::AuthQuery::FineGrainedPrivilege, std::vector<std::string>>>
-          &edge_type_privileges
-#endif
-      ) = 0;
+  virtual void DenyPrivilege(const std::string &user_or_role, const std::vector<AuthQuery::Privilege> &privileges) = 0;
```
Comment (gitbuda) [inline src/query/interpreter.hpp:115]: Also this?

File: src/glue/auth.cpp [hunk 3]
```diff
@@ -65,6 +65,8 @@ auth::Permission PrivilegeToPermission(query::AuthQuery::Privilege privilege) {
 auth::FineGrainedPermission FineGrainedPrivilegeToFineGrainedPermission(
     const query::AuthQuery::FineGrainedPrivilege fine_grained_privilege) {
   switch (fine_grained_privilege) {
+    case query::AuthQuery::FineGrainedPrivilege::NOTHING:
+      return auth::FineGrainedPermission::NO_PERMISSION;
```
Comment (Josipmrden) [inline src/glue/auth.cpp:69]: we can put here nothing as well

**Review by Josipmrden — CHANGES_REQUESTED** [review]
Check ShowPrivileges and add NOTHING there instead of NO_PERMISSION

**Review by gitbuda — APPROVED** [review]

**Review by Josipmrden — APPROVED** [review]

**Pull request merged**